app/src/pages/experiment/ExperimentCompareListPage.tsx

```diff
@@ -317,15 +335,41 @@ export function ExperimentCompareListPage() {
         accessorKey: "input",
         cell: ({ getValue }) => {
           const value = getValue() as string;
-          return <JSONText json={value} maxLength={50} />;
+          return (
+            <ContentPreviewTooltip content={value}>
+              <div
+                css={css`
+                  display: -webkit-box;
+                  -webkit-box-orient: vertical;
+                  -webkit-line-clamp: ${experiments.length};
+                  overflow: hidden;
+                `}
+              >
+                <JSONText json={value} disableTitle />
```

Comment:
nit: disableTitle makes it seem like the title is disabled, when in reality we probably just want `title` to be declarative as a prop